lib/kennel.rb

```diff
@@ -100,26 +101,30 @@ def parts_serializer
 
     def generated
       @generated ||= begin
-        Progress.progress "Generating" do
+        parts = Progress.progress "Finding applicable projects and parts" do
```

Comment:
says about the same ?
```suggestion
        parts = Progress.progress "Finding parts" do
```

Reply:
Sure, if you like